src/gsi/directDA_radaruse_mod.f90

```diff
@@ -0,0 +1,522 @@
+                                                ! mm**6 m**-3.
+
+      REAL(r_kind),PARAMETER :: pi=3.1415926 ! Pi.
+!     REAL(r_kind)           :: pi=ATAN(1.0)*4.0
+
+      REAL(r_kind),PARAMETER :: pipowf=7.0/4.0 ! Power to which pi is raised.
+
+      REAL(r_kind),PARAMETER :: N0r_0=8.0E+06 ! Intercept parameter in 1/(m^4) for rain.
+      REAL(r_kind),PARAMETER :: N0s_0=3.0E+06 ! Intercept parameter in 1/(m^4) for snow.
+      REAL(r_kind),PARAMETER :: N0g_0=4.0E+05 ! Intercept parameter in 1/(m^4) for graupel. (<=4.0E+6)
+      REAL(r_kind),PARAMETER :: N0h_0=4.0E+04 ! Intercept parameter in 1/(m^4) for hail.
+
+      REAL(r_kind),PARAMETER :: N0xpowf=3.0/4.0 ! Power to which N0r,N0s & N0h are
+                                                ! raised.
+
+      REAL(r_kind),PARAMETER :: approxpow_0=0.95 ! Approximation power for hail/graupel
+                                                 ! integral.
+
+      REAL(r_kind),PARAMETER :: rqrpowf=7.0/4.0 ! Power to which product rho * qr
+                                                ! is raised.
+      REAL(r_kind),PARAMETER :: rqsnpowf=7.0/4.0 ! Power to which product rho * qs
+                                                 ! is raised (dry snow).
+      REAL(r_kind),PARAMETER :: rqsppowf=7.0/4.0 ! Power to which product rho * qs
+                                                 ! is raised (wet snow).
+      REAL(r_kind),PARAMETER :: rqhnpowf=7.0/4.0 ! Power to which product rho * qh
+                                                 ! is raised (dry hail)
+!     REAL(r_kind),PARAMETER :: rqhppowf=(7.0/4.0)*approxpow ! Power to which product
+                                                             ! rho * qh is raised (wet hail).
+
+      REAL(r_kind),PARAMETER :: rhoi_0=917.  ! Density of ice (kg m**-3)
+      REAL(r_kind),PARAMETER :: rhor_0=1000. ! Density of rain (kg m**-3)
+      REAL(r_kind),PARAMETER :: rhos_0=100.  ! Density of snow (kg m**-3)
+      REAL(r_kind),PARAMETER :: rhoh_0=913.  ! Density of hail (kg m**-3)
+      REAL(r_kind),PARAMETER :: rhog_0=400.  ! Density of graupel (kg m**-3)
+
+      REAL(r_kind),PARAMETER :: rhoipowf=2.0     ! Power to which rhoi is raised.
+      REAL(r_kind),PARAMETER :: rhospowf=1.0/4.0 ! Power to which rhos is raised.
+      REAL(r_kind),PARAMETER :: rhoxpowf=7.0/4.0 ! Power to which rhoh is raised.
+
+      REAL(r_kind),PARAMETER :: Zefact=720.0 ! Multiplier for Ze components.
+
+      REAL(r_kind),PARAMETER :: lg10mul=10.0 ! Log10 multiplier
+
+!-----------------------------------------------------------------------
+! Declare local variables.
+!-----------------------------------------------------------------------
+      REAL(r_kind)           :: rqhppowf     ! Power to which product rho * qh is raised (wet hail).
+
+      REAL(r_kind) :: rcomp,scomp,hcomp,sumcomp
+      INTEGER(i_kind) :: i,j,k
+      REAL(r_kind) :: Zerf,Zesnegf,Zesposf,Zegf, Zehf
+      REAL(r_kind) :: Zehnegf,Zehposf
+
+      INTEGER(i_kind) :: ios
```

Comment:
The variable, ios, has been identified as unused.  Please remove unused variable.

Reply:
We removed them as you suggested.

https://github.com/NOAA-EMC/GSI/pull/142/files#diff-7de3a68f08280c2a8aff0eca26b15d2653a69cc4bcedb9608766da295bbee249R313